web/src/components/notification-pill/notification-pill.tsx

```diff
@@ -28,6 +28,9 @@ export default function NotificationPill({
       }
       style={{ opacity: show ? 1 : 0 }}
       onTransitionEnd={() => removeNotification(notification.id)}>
+      {notification.kind === 'pill' && notification.type === 'achievement' && (
+        <img src={require('../pages/dashboard/awards/star.svg')} alt="star" />
```

Comment:
Since the star icon doesn't have semantic meaning, we can skip the `alt` tag. From https://www.w3.org/WAI/tutorials/images/decorative: 

```
<img src={require('../pages/dashboard/awards/star.svg')} alt="" role="presentation" />
```

Reply:
Sure